data/sql/updates/pending_db_world/dustbelchernpcfix

```diff
@@ -0,0 +1 @@
+UPDATE `creature` SET `MovementType` = '1' WHERE `creature`.`guid` IN (6974,6989,7210); -- correcting MovementType for creatures Dustbelcher Mauler, Dustbelcher Wyrmhunter, and Dustbelcher Shaman
```

Comment:
The `MovementType` column has an integer datatype, so you don't need to put apostrophes around the value 1.

You also don't need to specify ``` `creature`.`guid` ``` here, just `guid` will work.

Otherwise looks good to me. 👍